### PR TITLE
Adjust styling for mobile version

### PR DESCRIPTION
**Pull request by bartoszskwara**

### CODE REVIEW

File: src/components/Navbar/Navbar.container.tsx [hunk 1]
```diff
@@ -64,15 +64,25 @@ const SidebarContainer = ({ sx, menuItems }: Props) => {
...
+            menuItems={menuItems}
+            logout={handleLogout}
+        />
+    ) : (
+        <NavbarView
             activePage={activePage}
-            setActivePage={setActivePage}
+            onPageSelect={handlePageSelect}
             menuItems={menuItems}
             logout={handleLogout}
```
Comment (mskwara) [inline src/components/Navbar/Navbar.container.tsx:78]: I think you could assign the proper component to a variable and then use it once with the props.
```
const NavbarComponent = mobile ? NavbarMobile : NavbarView;

<NavbarComponent ... ... ... />
```

File: src/components/Navbar/Navbar.view.tsx [hunk 2]
```diff
@@ -1,40 +1,26 @@
...
+import Styled from './Navbar.styled';
 
 interface Props {
-    sx?: SxProps<Theme>;
     activePage: string;
-    setActivePage: Dispatch<SetStateAction<string>>;
+    onPageSelect: (name: string) => void;
     menuItems: MenuItem[];
     logout: () => void;
+    className?: string;
```
Comment (mskwara) [inline src/components/Navbar/Navbar.view.tsx]: You could probably create Props once and export it for Mobile and View usages.

Comment (bartoszskwara) [reply src/components/Navbar/Navbar.view.tsx]: Done

File: src/components/Navbar/Navbar.container.tsx (hunk 1 above)
Comment (bartoszskwara) [reply src/components/Navbar/Navbar.container.tsx:78]: Done

File: src/components/Layout/Layout.tsx [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
 import menuItems from './menuItems';
 
 const Layout = () => (
-    <Box sx={{ display: 'flex', flex: 1 }}>
-        <Sidebar menuItems={menuItems} />
+    <Box
+        sx={(theme) => ({
+            display: 'flex',
+            flex: 1,
+            [theme.breakpoints.down('sm')]: {
```
Comment (mskwara) [inline src/components/Layout/Layout.tsx:12]: Shouldn't useIsMobile hook be used here?

**Review by mskwara — APPROVED** [review]

**Pull request merged**